### ISSUE TITLE
Catalog / DB to reside on a local network location -- usable by client computer(s) over the network.

### ISSUE THREAD

**Issue by bgri** [rubenwe/StlVault#8]
In my workflow I use multiple computers (and computer types (OSX, Linux, Win10)) to access files for different purposes (creation, editing, printing, etc). It would be nice to have access to a common database/catalogue from any of my computers to be able to add, update the catalogue. 

Perhaps just point each local client at the network drive location?

**Comment by rubenwe** [comment]
I'd love to add features around having multiple computers share a common database. Putting a database in a shared location is one of the ways this could be done. For the time beeing I'd need to put an additional machine identifier in the metadata because quite a bit of functionality depends on the models being available locally.

My workflow is also distributed between machines: I have a notebook that goes with me to the printer, but the main workstation stays where it is. So I've also been thinking about distributed workflows a bit. I'd love to have a shared database and to be able to transfer files on the go. Also: Backups etc.

As the name of the tool might imply - I'm planning on going in that direction sooner or later. I have a few features that are higher up on the priority queue - but I'll keep you updated!

Thanks for your feedback!

**Comment by bgri** [comment]
That's great news! Thank you for this.

Oh, did you want minor tweaks added here too as new issues? things like UX: add more facings in image rotation (every 45 deg. instead of 90). that sort of thing? Not what I'd call a primary feature, more of an enhancement.

**Comment by rubenwe** [comment]
@bgri Yeah, go ahead - if I don't like the request I can still close it 😄.

Smaller rotation angles make sense 👍 .
